docs/core/compatibility/deployment/7.0/deprecated-trimmer-default-action.md

```diff
@@ -9,15 +9,11 @@ The value of the `TrimmerDefaultAction` property is now ignored by the publish p
 
 ## Previous behavior
 
-Previously, only assemblies that were opted-in with `<IsTrimmable>true</IsTrimmable>` in the library project file were trimmed with the action specified by the `TrimmerDefaultAction`. The default value for that property was `link` in .NET 6.
+Previously, only assemblies that were opted-in with `<IsTrimmable>true</IsTrimmable>` in the library project file were trimmed with the action specified by the `TrimmerDefaultAction`. The default value for that property was `copy` in .NET 6. While apps with warnings were more likely to work with this behavior, they still couldn't be guaranteed to not affect behavior. In addition, this caused apps to be larger than if the entire app was trimmed.
```

Comment:
```suggestion
Previously, only assemblies that were opted-in with `<IsTrimmable>true</IsTrimmable>` in the library project file were trimmed with the action specified by `TrimmerDefaultAction`. In .NET 6, the default value for that property was `copy`. While apps with warnings were more likely to work when this action was specified, run-time behavior could still be affected. In addition, the `copy` action caused apps to be larger than if the entire app was trimmed.
```

Reply:
Trying to avoid the double negative.

Reply:
Couldn't commit this suggestion directly, but added it in a different commit.